plugins/system/fields/src/Extension/Fields.php

```diff
@@ -493,6 +501,13 @@ public function onContentPrepare(Content\ContentPrepareEvent $event)
         foreach ($fields as $key => $field) {
             $item->jcfields[$field->id] = $field;
         }
+
+        PluginHelper::importPlugin('fields');
+
+        /*
+        * Event allow plugins to modify the the fields it is with content prepare
+        */
+        Factory::getApplication()->triggerEvent('onCustomFieldsContentPrepare', [$context, $item, &$item->jcfields]);
```

Comment:
```suggestion
        $this->getApplication()->triggerEvent('onCustomFieldsContentPrepare', [$context, $item, &$item->jcfields]);
```
Hmm, seems I mossed one.